services/tasks/local.go

```diff
@@ -184,6 +184,11 @@ func (l *local) Create(ctx context.Context, r *api.CreateTaskRequest, _ ...grpc.
 			Options: m.Options,
 		})
 	}
+	if strings.HasPrefix(container.Runtime.Name, "io.containerd.runtime.v1.") {
+		log.G(ctx).Warn("runtime v1 is deprecated in containerd v1.4, and planned to be removed in containerd v2.0")
```

Comment:
Mentioning `v2.0` in the logs here doesn't feel right to me. This makes it sound like there is active 2.0 planning.

I think with these sort of logs, it is better to be more prescriptive. Say the runtime is deprecated, and what to do about it. Either upgrade to another one, or check out a doc. I think upgrade is the right thing to say here along with the version to upgrade to if possible.

Reply:
updated